.github/commands/release-blocker.yml

```diff
@@ -0,0 +1,46 @@
+---
+trigger: release-blocker
+title: Release Blocker
+description: Describe the issue(s) preventing a release from being published
+surfaces:
+  - issue
+steps:
+  - type: form
+    style: embedded
+    body:
+      - type: textarea
+        attributes:
+          id: blocker-description
+          label: Description
+          placeholder: Describe the issue that prevents the release from being published.
+
+      - type: dropdown
+        attributes:
+          id: blocker-type
+          label: Type
+          options:
+            - 'Component or type removed'
+            - 'Type incompatability'
+            - 'Other'
+
+      - type: input
+        id: other
+        attributes:
+          label: Other
+          description: 'Provide a custom blocker type if other was used above'
```

Comment:
Oo I love this too! 🔥 So what is the game plan to keep this list up-to-date? 🤔 

Are we looking into adding the new blocker types when they were specified in "other" and update the list here or do you have something different in your mind?

Reply:
@broccolinisoup honestly nothing specific came to mind, super open to ideas here. Figured it could be a manual list that we update as things come up and if we find ourselves using this command a lot then we could try and keep track of what gets put into the "other" field and use that data to go off of (but I honestly have no idea what's best here haha)

Reply:
That sounds good to me! We can try to see what works best really. I am perfectly happy to start manual 💪🏻